src/components/Books/Book/AddBook.js

```diff
@@ -1,11 +1,41 @@
 import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../../redux/books/books';
 
-const AddBook = () => (
-  <form>
-    <input type="text" name="Title" placeholder="Title" />
-    <input type="text" name="Author" placeholder="Author" />
-    <input type="button" value="Add" />
-  </form>
-);
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const AddBookHandler = () => {
+    const newBook = {
+      id: uuidv4(),
+      title: document.getElementById('title').value,
+      author: document.getElementById('author').value,
+    };
+    // dispatch an action and pass it the newBook object (your action's payload)
+    dispatch(addBook(newBook));
+
+    document.getElementById('title').value = '';
+    document.getElementById('author').value = '';
+  };
+
+  return (
+    <form>
+      <input
+        id="title"
+        type="text"
+        name="Title"
+        placeholder="Title"
+      />
+      <input
+        id="author"
+        type="text"
+        name="Author"
+        placeholder="Author"
+      />
+      <input type="button" value="Add" onClick={AddBookHandler} />
+    </form>
+  );
```

Comment:
- [ ] Let us validate our inputs so that a user couldn't add the empty book title and author. 
![user-4](https://user-images.githubusercontent.com/55635977/139801120-c237ac81-e56d-49f6-ae10-70d8a221ff31.png)